docs/containers-storage.conf.5.md

```diff
@@ -34,6 +34,19 @@ The `storage` table supports the following options:
   container storage graph dir (default: "/var/lib/containers/storage")
   Default directory to store all writable content created by container storage programs.
 
+**rootless_storage_path**="$HOME/.local/share/containers/storage"
```

Comment:
What is the RHS of the `=` supposed to be? In all other options it's simply an empty string `""`. Why is this one different? Especially why is it misleadingly different? The `=` here implies that it's a default, but right below it says otherwise.

Reply:
the value of rootless_storage_path will not be used unless the user overwrites it in storage.conf. I think it can be described as "". But I'n not sure. I need ask @rhatdan